File: 백준/Bronze/10807. 개수 세기/개수 세기.c
int main()
{
    int n;
    scanf("%d", &n);

    int a[n];

    for(int i=0; i<n; i++){
        scanf("%d", &a[i]);
    }
    getchar();

    int count = 0;
    int v;
    scanf("%d", &v);

    for(int j=0; j<n; j++){
        if(a[j] == v)
            count++;
    }

    printf("%d", count);

    return 0;
}

File: 백준/Bronze/10807. 개수 세기/개수 세기.js
let fs = require("fs");
let input = fs.readFileSync("/dev/stdin").toString().split("\n");

let n = Number(input[0]); // 정수의 개수 
let arr = input[1].split(" ").map(Number);
let v = Number(input[2]); // 찾으려는 정수

let count = 0;
for (let i = 0; i < n; i++) {
  if (arr[i] == v)
    count++;
}

console.log(count);